checks/E1F2C3.yaml

```diff
@@ -0,0 +1,79 @@
+id: "E1F2C3"
+name: SAPHanaController roles correctly configured
+group: Pacemaker
+description: |
+  SAPHanaController roles correctly configured
+remediation: |
+  ## Abstract
+  SAPHanaController roles should be configured as "Promoted" and "Unpromoted" and not as "Master" and "Slave".
+
+  ## Remediation
+
+  Edit the crm configuration and change the values of roles as per the best parctices.
```

Comment:
Typo: `parctices` -> `practices`
By the way, I would remove the new line after `Remediation` header, we don't have it on other checks